pkg/cloud/cloud.go

```diff
@@ -102,6 +102,10 @@ type DiskOptions struct {
 	VolumeType       string
 	IOPSPerGB        int
 	AvailabilityZone string
+	Encrypted        bool
+	// fully qualified resource name to the key to use for encryption.
+	// example: arn:aws:kms:us-east-1:012345678910:key/abcd1234-a123-456a-a12b-a123b4cd56ef
+	KmsKeyID string
```

Comment:
In case of `KmsKeyID` is empty, EBS will use a default key to create the volume. See: https://docs.aws.amazon.com/sdk-for-go/api/service/ec2/#CreateVolumeInput

Reply:
See above comment